widget/embedded/src/components/Quote/QuoteCostDetails.tsx

```diff
@@ -0,0 +1,221 @@
+                            fee.amount,
+                            GAS_FEE_MIN_DECIMALS,
+                            GAS_FEE_MAX_DECIMALS
+                          )}{' '}
+                          {fee.asset.symbol} ($
+                          {numberToString(
+                            usdValue,
+                            USD_VALUE_MIN_DECIMALS,
+                            USD_VALUE_MAX_DECIMALS
+                          )}
+                          )
+                        </Typography>
+                      </Tooltip>
+                    </FeeSection>
+                    <Divider size={8} />
+                  </>
+                )
+              );
+            })
+          )}
+          <Divider size={12} />
+          <FeeSection>
+            <Typography variant="label" size="medium">
+              {i18n.t('Total Payable Fee')}
+            </Typography>
+            <Typography variant="label" size="medium">
+              $
+              {numberToString(
+                totalNetworkFeeInUsd,
+                USD_VALUE_MIN_DECIMALS,
+                USD_VALUE_MAX_DECIMALS
+              )}
+            </Typography>
+          </FeeSection>
+          <Divider size={12} />
+          <Line />
+          <Divider size={12} />
+          <CustomCollapsible
+            triggerAnchor="bottom"
+            onClickTrigger={() => setOpenCollapse((prev) => !prev)}
+            trigger={
+              <>
+                <Typography size="small" variant="body" color="neutral700">
+                  {openCollapse
+                    ? i18n.t('Hide non-payable fees')
+                    : i18n.t('Show non-payable fees')}
+                </Typography>
+                <Divider size={4} direction="horizontal" />
+                <ExpandedIcon orientation={openCollapse ? 'up' : 'down'}>
+                  <ChevronDownIcon size={12} color="gray" />
+                </ExpandedIcon>
+              </>
+            }
+            open={openCollapse}>
+            <Typography size="small" variant="title">
+              {i18n.t('Description')}
+            </Typography>
+            <Divider size={4} />
+            <Typography size="small" variant="body" color="neutral700">
+              {i18n.t(`The following fees are considered in the transaction output and
+                you won’t need to pay extra gas for them.`)}
+            </Typography>
+            <Divider size={12} />
+            {NON_PAYABLE_FEES.map((item, index) => {
+              const totalFeeInUsd = getTotalSwapsFeeInUsd(swaps, item.name);
+              const key = `fee_${index}`;
+              return (
+                <NonPayableFee
+                  key={key}
+                  fee={totalFeeInUsd}
+                  label={i18n.t(item.lable)}
+                />
+              );
+            })}
+            <Divider size={4} />
+            <Line />
+            <Divider size={12} />
+          </CustomCollapsible>
```

Comment:
I believe it would be beneficial to include a condition: if a swap doesn't have a non-payable fee, we should refrain from displaying this collapsible component.

Reply:
fixed